docusaurus/docs/iOS/02-tutorials/03-livestream.mdx

```diff
@@ -5,328 +5,325 @@ description: How to build a livestream experience using Stream's video SDKs
+                        Text("Go Live")
                     }
                 } else {
                     Button {
                         Task {
-                            loading = true
-                            try await call.startBroadcasting()
+                            try await call.stopLive()
                         }
                     } label: {
-                        Text("Start stream")
+                        Text("Stop Livestream")
                     }
                 }
-                
-                Button {
-                    onLeaveCall()
-                } label: {
-                    Text("Leave call")
-                }
-            }
-            .opacity(loading ? 0 : 1)
-            
-            ProgressView()
-                .opacity(loading ? 1 : 0)
-        }
-        .padding()
-    }
-    .task {
-        for await videoEvent in streamVideo.subscribe() {
-            switch videoEvent {
-            case .typeCallBroadcastingStartedEvent:
-                self.isBroadcasting = true
-                self.isLoading = false
-            case .typeCallBroadcastingStoppedEvent:
-                self.isBroadcasting = false
-                self.isLoading = false
-            default:
-                break
             }
+            .padding()
         }
+        .background(Color(UIColor.systemBackground))
+        .navigationBarHidden(true)
     }
+    
 }
 ```
 
-We take the participants from the call object, and we sort them with the default sort comparators:
+Upon running your app, you will be greeted with an interface that looks like this:
 
-```swift
-private var participants: [CallParticipant] {
-    return call.participants.map(\.value).sorted(using: defaultComparators)
-}
+![Livestream](../assets/tutorial-livestream.png)
+
+Stream uses a technology called SFU cascading to replicate your livestream over different SFUs around the world.
+This makes it possible to reach a large audience in realtime.
+
+Now let's press **Go live** in the iOS app and click the link below to watch the video in your browser.
+
+<TokenSnippet sampleApp='livestream' displayStyle='join' />
+
+Let's take a moment to review the SwiftUI code above. `Call.state` exposes all the `@Published` objects you need.
+
+The most important ones are:
+
+```
+call.state
+call.state.participants
 ```
 
-Then, we present the first participant in a `VideoCallParticipantView`, which shows the camera feed of that user. You can build a different UI here, based on the requirements of your app:
+The [participant state docs](../03-guides/03-call-and-participant-state.mdx) show all the available data.
+
+The livestream layout is built using standard SwiftUI. The [VideoRenderer](../04-ui-components/02-video-renderer.mdx) component is provided by Stream.
+**VideoRenderer** renders the video and a fallback. You can use it for rendering the local and remote video.
+
+### Step 4 - (Optional) Publishing RTMP using OBS
```

Comment:
```suggestion
### Step 5 - (Optional) Publishing RTMP using OBS
```

Reply:
skipping to keep the same structure with Android tutorial